requirements.txt

```diff
@@ -17,6 +17,11 @@ pyngrok==2.1.4
 # Web
 ## http server
 flask==1.1.2
+## flask user authentication management
+flask-login==0.5.0
+## user form validators
+WTForms==2.3.1
```

Comment:
We really need these two requirements about `user form validators` ?

Reply:
Flask-WTF is the flask implementation of WTForm which is a widely used python form validator, to properly handle login forms, we need a serious validator.

Flask-WTF is enough, we could just add it to this file (WTForm is a Flask-WTF requirement) but I think it's better to freeze both versions in order to maximise security (Flask-WTF doesn't specify a WTForm version)